src/hydra-queue-runner/hydra-queue-runner.cc

```diff
@@ -39,21 +39,25 @@ std::string getEnvOrDie(const std::string & key)
 }
 
 
-State::State(uint16_t metricsPort)
+State::State(std::optional<uint16_t> metricsPortOpt)
     : config(std::make_unique<HydraConfig>())
     , maxUnsupportedTime(config->getIntOption("max_unsupported_time", 0))
     , dbPool(config->getIntOption("max_db_connections", 128))
     , maxOutputSize(config->getIntOption("max_output_size", 2ULL << 30))
     , maxLogSize(config->getIntOption("max_log_size", 64ULL << 20))
     , uploadLogsToBinaryCache(config->getBoolOption("upload_logs_to_binary_cache", false))
     , rootsDir(config->getStrOption("gc_roots_dir", fmt("%s/gcroots/per-user/%s/hydra-roots", settings.nixStateDir, getEnvOrDie("LOGNAME"))))
-    , metricsPort(config->getIntOption("queue_runner_metrics_port", metricsPort))
+    , metricsPort(config->getIntOption("queue_runner_metrics_port", 9099))
```

Comment:
Is there a "better" default port to use?

Reply:
Let's use 9198, since hydra-notify uses 9199.